Framework/DataHandling/test/ISISJournalTest.h

```diff
@@ -0,0 +1,216 @@
+// Mantid Repository : https://github.com/mantidproject/mantid
+//
+// Copyright &copy; 2020 ISIS Rutherford Appleton Laboratory UKRI,
+//   NScD Oak Ridge National Laboratory, European Spallation Source,
+//   Institut Laue - Langevin & CSNS, Institute of High Energy Physics, CAS
+// SPDX - License - Identifier: GPL - 3.0 +
+#pragma once
+
+#include <cxxtest/TestSuite.h>
+
+#include "MantidDataHandling/ISISJournal.h"
+
+using namespace Mantid::DataHandling::ISISJournal;
+
+class ISISJournalTest : public CxxTest::TestSuite {
+public:
+  // This pair of boilerplate methods prevent the suite being created statically
+  // This means the constructor isn't called when running other tests
+  static ISISJournalTest *createSuite() { return new ISISJournalTest(); }
```

Comment:
You don't need these two lines since we have no constructor in this test class, it's something which gets copied + pasted round our codebase a lot for no reason :P